jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/Configuration.java

```diff
@@ -17,6 +17,7 @@
 import java.util.Collections;
 import java.util.ServiceLoader;
 
+import org.eclipse.jetty.util.ClassMatcher;
```

Comment:
This changes the signatures of methods using `ClassMatcher` as input or return.
Also, the naming change of server/system to hidden/protected hasn't been performed here yet

Reply:
I don't want to change ee9 naming more than necessary.